### DISCUSSION TITLE
Is airyx a fork of FreeBSD?

### DISCUSSION THREAD

**Discussion by z4kz** [ravynsoft/ravynos#225] [Q&A] [answered]
Hello,

Is airyx a full fork of the FreeBSD codebase or will you continue to receive updates from upstream?

**Comment by z4kz** [comment, marked as answer]
I ask because it seems that Mach support is being added to the kernel.

**Reply by mszoek** [reply]
It's a soft fork. I track the upstream repository and incorporate all changes. We are currently tracking 13.0-STABLE (git branch stable/13).
